README.md

```diff
@@ -12,7 +12,78 @@ This is the main repository, which contains the majority of the source code for
+To have a complete overview about Xmipp please visit the [documentation web](https://i2pc.github.io/docs/). The recommended way for users (not developers) to install and use Xmipp is via the [Scipion](http://scipion.i2pc.es/) framework, where you can use Xmipp with other Cryo-EM-related software. 
+
+### As a Scipion plugin (Recommended)
+The Scipion installer should take care of all dependencies for you by default, except for CUDA. To correctly interoperate with CUDA, please visit the official [installation guide](https://scipion-em.github.io/docs/release-3.0.0/docs/scipion-modes/how-to-install.html#installation) of Scipion.
+
+Once Scipion is in working order, launch the Xmipp Plugin installer with
+
+`scipion3 installp -p scipion-em-xmipp`
+
+By default this will employ all CPU cores available in the system for compilation. If this is not the desired behavior, please consider using `-j <N>` flag to limit the amount of CPU cores.
+
+### As a Scipion plugin with pre-installed dependencies
+The former method also installs dependencies (including the compiler) though conda. If you prefer to use externally installed dependencies and avoid downloading them from conda use
+
+`scipion3 installp -p scipion-em-xmipp --noBin`
+
+`scipion3 installb xmippSrc`
+
+Mind that for this command to succeed, all dependencies need to be pre-installed and findable by Xmipp. When using non-standard install directories, this needs to be indicated to Xmipp though the Scipion configuration file. See Software dependency section for further details.
+
+### Developer installation with Scipion
+For developers, it becomes handy to install Xmipp in an external directory for easy integration with IDEs. By default Xmipp installer takes care of installing dependencies though conda and linking to Scipion.
+
+The first step is to download Xmipp sources:
+
+`git clone https://github.com/I2PC/xmipp.git`
+
+`cd xmipp`
+
+Then all child repositories need to be fetched, with an optional git branch parameter. For repositories where the branch does not exist, devel is utilized.
+
+`./xmipp getSources [-b branch]`
+
+Install the Scipion plugin and dependencies
+
+`scipion3 installp -p src/scipion-em-xmipp --devel`
+
+Compile Xmipp
+
+`scipion3 run ./xmipp`
```

Comment:
It is needed for the subsequent `scipion3 installp -p src/scipion-em-xmipp --devel`, as otherwise the plugin repo would not be available. `scipion3 installp -p src/scipion-em-xmipp --devel` should be run before `./xmipp` such that dependencies are installed though conda.

Reply:
In case dependencies are pre-installed, `./xmipp getSources [-b branch]` could be omitted and `scipion3 installp -p src/scipion-em-xmipp --devel --noBin` could be run at the end

Reply:
> It is needed for the subsequent `scipion3 installp -p src/scipion-em-xmipp --devel`, as otherwise the plugin repo would not be available. `scipion3 installp -p src/scipion-em-xmipp --devel` should be run before `./xmipp` such that dependencies are installed though conda.

Okay, it was the "dependencies installed through conda" what i was missing. In such case, yeah, the commands are correctly placed.